src/Components/Body/BodyHeader.js

```diff
@@ -0,0 +1,22 @@
+import React, {useState} from "react"
+// import { Jumbotron, Container} from "reactstrap;"
+
+const BodyHeader = ({title}) => {
+    // set up state for the likes
+    // const [title, setTitle] =useState(props.title);
+    // setTitle(props.title)
+    console.log(title)
+    
+    return (
+    <div>
+        {/* <Jumbotron fluid>
+            <Container fluid> */}
+                <h1 className="display-3"> NASA Photo of the Day</h1>
+                <p className="BodyHeader">{title.title}</p>
```

Comment:
may want to consider organizing props more clearly so that you don't end up with code such as "title.title". My project from when I did this looks similar though.

I like how you worked hard to separate everything into components